openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/DatabaseSchemaVersionPage.spec.ts

```diff
@@ -62,62 +76,74 @@ describe(
+          // Create Database
           cy.request({
-            method: 'PATCH',
-            url: `/api/v1/databaseSchemas/${databaseSchemaId}`,
-            headers: {
-              Authorization: `Bearer ${token}`,
-              'Content-Type': 'application/json-patch+json',
-            },
-            body: [
-              ...COMMON_PATCH_PAYLOAD,
-              {
-                op: 'add',
-                path: '/domain',
-                value: {
-                  id: domainId,
-                  type: 'domain',
-                  name: DOMAIN_CREATION_DETAILS.name,
-                  description: DOMAIN_CREATION_DETAILS.description,
+            method: 'POST',
+            url: `/api/v1/databases`,
+            headers: { Authorization: `Bearer ${token}` },
+            body: DATABASE_DETAILS_FOR_VERSION_TEST,
+          }).then((response) => {
+            data.database = response.body;
+
+            // Create Database Schema
+            cy.request({
+              method: 'PUT',
+              url: `/api/v1/databaseSchemas`,
+              headers: { Authorization: `Bearer ${token}` },
+              body: DATABASE_SCHEMA_DETAILS_FOR_VERSION_TEST,
+            }).then((response) => {
+              data.schema = response.body;
+
+              cy.request({
+                method: 'PATCH',
+                url: `/api/v1/databaseSchemas/${data.schema.id}`,
+                headers: {
+                  Authorization: `Bearer ${token}`,
+                  'Content-Type': 'application/json-patch+json',
                 },
-              },
-            ],
+                body: [
+                  ...COMMON_PATCH_PAYLOAD,
+                  {
+                    op: 'add',
+                    path: '/domain',
+                    value: {
+                      id: data.domain.id,
+                      type: 'domain',
+                      name: DOMAIN_CREATION_DETAILS.name,
+                      description: DOMAIN_CREATION_DETAILS.description,
+                    },
+                  },
+                ],
+              });
+            });
           });
         });
       });
     });
 
     after(() => {
       cy.login();
-      cy.getAllLocalStorage().then((data) => {
-        const token = getToken(data);
+      cy.getAllLocalStorage().then((responseData) => {
+        const token = getToken(responseData);
         cy.request({
           method: 'DELETE',
           url: `/api/v1/domains/name/${DOMAIN_CREATION_DETAILS.name}`,
           headers: { Authorization: `Bearer ${token}` },
         });
+
+        // Delete created user
+        cy.request({
+          method: 'DELETE',
+          url: `/api/v1/users/${data.user.id}?hardDelete=true&recursive=false`,
+          headers: { Authorization: `Bearer ${token}` },
+        });
+
+        hardDeleteService({
+          token,
+          serviceFqn: serviceDetails.serviceName,
+          serviceType: SERVICE_CATEGORIES.DATABASE_SERVICES,
+        });
```

Comment:
You can use [this](https://github.com/open-metadata/OpenMetadata/blob/a3689cad54784f2c66edabc2268e618abf9333db/openmetadata-ui/src/main/resources/ui/cypress/common/Entities/DatabaseClass.ts#L81) method